### PR TITLE
Action format

### PR DESCRIPTION
**Pull request by wyb1**
**What this PR does / why we need it**:

Add `gofmt` to github actions.

### CODE REVIEW

File: connectivity-exporter/internal/tools/tools.go [hunk 1]
```diff
@@ -5,11 +5,11 @@
 // Do not build this file by default, it's only parsed by go mod to
 // pull the dependencies.
 
+//go:build tools
 // +build tools
```
Comment (alban) [inline connectivity-exporter/internal/tools/tools.go:9]: What's the difference between `+build` and `go:build`?

Comment (wyb1) [reply connectivity-exporter/internal/tools/tools.go:9]: Starting with go1.17 it is preferred over `+build` 
https://golang.org/doc/go1.17

Comment (wyb1) [reply connectivity-exporter/internal/tools/tools.go:9]: Here are the referenced docs: https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md

Comment (istvanballok) [reply connectivity-exporter/internal/tools/tools.go:9]: Note: together with @wyb1 we found out that `gofmt` that is shipped with `go 1.17` is suggesting these changes.
This is a good hint for upgrading Go from 1.16 to 1.17.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -104,6 +104,15 @@ jobs:
         rm -rf /tmp/.buildx-cache
         mv /tmp/.buildx-cache-new /tmp/.buildx-cache
 
+    - name: Format Go code
+      run: |
+          OUTPUT=$(gofmt -l ./connectivity-exporter)
+          if [ -n "$OUTPUT" ]; then
+            echo "Please format the following files using gofmt"
+            echo "${OUTPUT}"
```
Comment (istvanballok) [inline .github/workflows/build.yml:112]:
```suggestion
            echo "${OUTPUT}"
            gofmt -d ./connectivity-exporter
```

Idea: show the diff in the logs of the github action.

**Review by istvanballok — APPROVED** [review]

**Pull request merged**